packaging/mingw-w64/travis-before-install.sh

```diff
@@ -20,9 +20,16 @@ case $TRAVIS_OS_NAME in
         # to zstd instead of xz compression
         $msys2 pacman -Sy --noconfirm pacman
         choco upgrade --no-progress -y msys2
-        $msys2 pacman --sync --noconfirm --needed mingw-w64-x86_64-toolchain mingw-w64-x86_64-cmake mingw-w64-x86_64-openssl mingw-w64-x86_64-cyrus-sasl
 
         ## Install more MSYS2 packages from https://packages.msys2.org/base here
+        $msys2 pacman --sync --noconfirm --needed mingw-w64-x86_64-toolchain mingw-w64-x86_64-cmake mingw-w64-x86_64-openssl mingw-w64-x86_64-lz4 mingw-w64-x86_64-zstd
```

Comment:
add what you need (zstd?) but dont remove sasl

Reply:
Cyrus SASL isn't used on Windows (https://github.com/edenhill/librdkafka/blob/master/src/rdkafka_sasl.c#L363-L367), so there's no benefit in installing it, right?

Reply:
Good point :+1: